[2023-09-10T16:09:26.644+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: website_crawler.create_data_table manual__2023-09-10T16:09:04.573370+00:00 [queued]>
[2023-09-10T16:09:26.681+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: website_crawler.create_data_table manual__2023-09-10T16:09:04.573370+00:00 [queued]>
[2023-09-10T16:09:26.686+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-09-10T16:09:26.747+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_data_table> on 2023-09-10 16:09:04.573370+00:00
[2023-09-10T16:09:26.771+0000] {standard_task_runner.py:57} INFO - Started process 57 to run task
[2023-09-10T16:09:26.795+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'website_crawler', 'create_data_table', 'manual__2023-09-10T16:09:04.573370+00:00', '--job-id', '488', '--raw', '--subdir', 'DAGS_FOLDER/crawl_data_log_into_postgres.py', '--cfg-path', '/tmp/tmpni_6vl14']
[2023-09-10T16:09:26.810+0000] {standard_task_runner.py:85} INFO - Job 488: Subtask create_data_table
[2023-09-10T16:09:27.227+0000] {task_command.py:415} INFO - Running <TaskInstance: website_crawler.create_data_table manual__2023-09-10T16:09:04.573370+00:00 [running]> on host 30cb2bd58730
[2023-09-10T16:09:27.920+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='website_crawler' AIRFLOW_CTX_TASK_ID='create_data_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-10T16:09:04.573370+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-10T16:09:04.573370+00:00'
[2023-09-10T16:09:27.949+0000] {sql.py:274} INFO - Executing: -- create data table
CREATE TABLE IF NOT EXISTS test2(
    title CHAR(5000) NOT NULL,
    author CHAR(5000) NOT NULL,
    date CHAR(5000) NOT NULL,
    time CHAR(5000) NOT NULL,
    summary CHAR(5000) NOT NULL);
[2023-09-10T16:09:28.006+0000] {base.py:73} INFO - Using connection ID 'postgres_local' for task execution.
[2023-09-10T16:09:28.070+0000] {base.py:73} INFO - Using connection ID 'postgres_local' for task execution.
[2023-09-10T16:09:28.092+0000] {sql.py:418} INFO - Running statement: -- create data table
CREATE TABLE IF NOT EXISTS test2(
    title CHAR(5000) NOT NULL,
    author CHAR(5000) NOT NULL,
    date CHAR(5000) NOT NULL,
    time CHAR(5000) NOT NULL,
    summary CHAR(5000) NOT NULL);, parameters: None
[2023-09-10T16:09:28.152+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=website_crawler, task_id=create_data_table, execution_date=20230910T160904, start_date=20230910T160926, end_date=20230910T160928
[2023-09-10T16:09:28.288+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-10T16:09:28.400+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
